src/main/java/cs2103/v15_1j/jimjim/JJStorage.java

```diff
@@ -0,0 +1,79 @@
+package cs2103.v15_1j.jimjim;
+
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.List;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.reflect.TypeToken;
+import java.lang.reflect.Type;
+import java.nio.file.Files;
+
+class JJStorage implements Storage {
+	private File saveFile;
+	private List<TaskEvent> list;
+	private Type listOfTaskEventType;
+	private GsonBuilder builder;
+	private Gson gson;
+	
+	public JJStorage() {
+		listOfTaskEventType = new TypeToken<List<TaskEvent>>(){}.getType();
+		builder = new GsonBuilder();
+		gson = builder.create();
+	}
+	
+	// Sets save file to be used for saving/loading
+	public void setSaveFile(String fileName) {
+		saveFile = new File(fileName);
+	}
+	
+	// Return save file
+	public File getSaveFile() {
+		return saveFile;
+	}
+
+	@Override
+	public List<TaskEvent> load() throws IOException {
```

Comment:
If you don't want it to throw an IOException (like me :P - a just prefer exception to be caught as early as possible) can return null if there are some errors.